ObjectiveSee/RansomWhere.pkg.recipe

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads, unzips and verifies the signature for the installer of the latest version of RansomWhere? by Objective-See.</string>
+    <key>Identifier</key>
+    <string>com.github.its-unibas.pkg.RansomWhere</string>
+    <key>MinimumVersion</key>
+    <string>0.6.0</string>
+    <key>Input</key>
+    <dict>
+        <key>NAME</key>
+        <string>RansomWhere</string>
+    </dict>
+    <key>ParentRecipe</key>
+    <string>com.github.its-unibas.download.RansomWhere</string>
+    <key>Process</key>
+    <array>
+        <dict>
+            <key>Arguments</key>
+            <dict>
+                <key>app_path</key>
+                <string>%RECIPE_CACHE_DIR%/%NAME%/%NAME%_Installer.app</string>
```

Comment:
Same comments as above apply.

Reply:
I have now implemented the requested changes